Modules/MDEModule/readme.md

```diff
@@ -1,88 +1,86 @@
-﻿# Get-MDEUsersDevicesRiskScore
+﻿# Get-MDEInsight
 
 ## Description
-This module will check the risk score in Microsoft Defender for Endpoint of all the machines on which a user logged on interactively.
+This module will return the risk score and exposure level from Microsoft Defender for Endpoint of all the machines on which a user logged on interactively and for all machines with specified IP addresses.
 
 ## Suported Entity Types
-* Account (ideally with a userSid property, if the userSid will be looked up thanks to the other properties)
+* Accounts (if the accounts have objectSid available)
```

Comment:
should we consider other language here since.  for example (If entity contains objectSid or objectSid can be obtained from Microsoft Graph)